crates/router/src/connector/bambora.rs

```diff
@@ -486,72 +552,78 @@ impl ConnectorIntegration<api::Authorize, types::PaymentsAuthorizeData, types::P
 
     fn get_url(
         &self,
-        _req: &types::PaymentsAuthorizeRouterData,
+        req: &types::PaymentsCancelRouterData,
         connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Ok(format!("{}{}", self.base_url(connectors), "/v1/payments"))
+        let connector_payment_id = req.request.connector_transaction_id.clone();
+        Ok(format!(
+            "{}/v1/payments/{}{}",
+            self.base_url(connectors),
+            connector_payment_id,
+            "/void"
```

Comment:
Not required as a variable
```suggestion
        Ok(format!(
            "{}/v1/payments/{}/void",
            self.base_url(connectors),
            connector_payment_id,
```

Reply:
Done